packages/compass-global-writes/src/components/common-styles.ts

```diff
@@ -0,0 +1,19 @@
+import { css, spacing } from '@mongodb-js/compass-components';
+
+export const bannerStyles = css({
+  textAlign: 'justify',
```

Comment:
this adds extra space in the banner and the message looks a bit weird (in the screenshot attached in the PR description). also, would be nice if we can split the error message by new line (server error has it).

Reply:
changed this setting for the error specifically. otherwise most messages I tried looked better with justify. also added pre-wrap on the error.